.travis.yml

```diff
@@ -55,10 +54,14 @@ script:
   - stty cols $COLUMNS rows 40
   - tput cols; stty size
   - pushd inst
-  - octave --path=/home/travis/build/cbm755/octsympy/pytave -W --no-gui --eval "r = octsympy_tests; exit(r)"
+  - if [ "x$PYTAVE" = "xyes" ]; then
+        octave --path=$TRAVIS_BUILD_DIR/pytave -W --no-gui --eval "sympref ipc native; r = octsympy_tests; exit(r)";
+    else
+        octave -W --no-gui --eval "r = octsympy_tests; exit(r)";
+    fi
```

Comment:
I have a slight preference for the if-else form, but not a big deal for me either way.

Doctest runner will need this sympref bit too, eventually